SFBaseKitDemo/SFBaseKitDemo/ExampleHomeModule/HomeCoordinator.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  HomeCoordinator.swift
+//  SFBaseKitDemo
+//
+//  Created by Aleksandar Geyman on 21.04.20.
+//  Copyright © 2020 Upnetix. All rights reserved.
+//
+
+import SFBaseKit
+
+protocol HomeSceneDelegate: Coordinator {
+    
+}
+
+class HomeCoordinator: Coordinator {
+    
+    unowned private var navController: UINavigationController!
+    
+    override func start() {
+        
+        // Instantiate Home Screen and add scene delegate.
+        let rootVC = HomeViewController.instantiateFromStoryboard()
+        rootVC?.sceneDelegate = self
+        
+        // As login flow is finished new navigation flow is started.
+        let navController = UINavigationController(rootViewController: rootVC ?? UIViewController())
```

Comment:
We discussed that - line 25 to 29 should be moved inside of AppCooridnator.

Reply:
Done.